protractor.conf.js

```diff
@@ -21,13 +21,13 @@ const width = 1366;
 const height = 768;
 
 const SAVE_SCREENSHOT = process.env.SAVE_SCREENSHOT === 'true';
-const API_CONTENT_HOST = process.env.API_CONTENT_HOST || 'http://localhost:8080';
+const APP_CONFIG_ECM_HOST = process.env.APP_CONFIG_ECM_HOST || 'http://localhost:8080';
```

Comment:
Shouldn't this be just API_HOST like it is on adw and apps?

Reply:
We are still discussing that. For now, there are different variables for ecm and bpm.

Reply:
@popovicsandras am I correct ?

Reply:
@adinapitul We are changing and standardising these variables in every repository at the moment. What you can see here is going to be the new name of variables, which is going to be accross every repo, and with this we even get rid of the variable name differences between local dev and deployement time. (We are going to use the same var names as what the docker container needs)